src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    result = result - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    result = result - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
try to make it work without `if else` - [checklist ](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)point 4, 5